### PR TITLE
Contact validation

### PR DESCRIPTION
**Pull request by SabiMantock**
Main features:

- Added `id` attributes to `form` and `input` tags
- Added an error message
- Added styling to the error message
- Added form validation with javascript

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -70,6 +73,16 @@ function closeMobileMenu() {
...
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+
+  if (email.value !== email.value.toLowerCase()) {
+    error.textContent = 'Please the content of the email field has to be in lower case.';
+  } else {
+    error.textContent = '';
+  }
+});
+
```
Comment (mckent05) [inline script.js:85]: - According to the project requirements, If a user submits the contact form with the email field containing only lowercase characters the form is sent normally. Your project doesn't submit when the email field is valid. I suggest you add `form.submit()` in your `else` block before `error.textContent=''`.  Please fix this 🙏 

Your project:
hffff

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @SabiMantock,

# STATUS: CHANGES REQUIRED ♻️ ♻️  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- GitHub flow was rightly used 👍 👍 
- PR is professional 👏 👏 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (SabiMantock) [reply script.js:85]: Thank you for the review

**Review by Munsa1 — APPROVED** [review]
Hi @SabiMantock, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**